README.md

```diff
@@ -173,6 +173,16 @@ for Terraform version 0.11 and below.
    }
    ```
 
+   **Venafi as a Service for EU**:
+
+   ```text
+   provider "venafi" {
+     url     = "https://api.eu.venafi.cloud"
```

Comment:
Resolve if you are sure we need to use `api.eu.venafi.com` (as opposed to `api.venafi.eu`).

Reply:
It was determined we should use `api.venafi.eu`. The PR is updated to that.